engines/tetraedge/detection.cpp

```diff
@@ -45,6 +45,7 @@ static const Common::Language *getGameLanguages() {
 		Common::IT_ITA,
 		Common::ES_ESP,
 		Common::RU_RUS,
+		Common::HE_ISR,
```

Comment:
It would be good to note it's an optional fan translation in a comment here.

For the other languages, the data is in the standard game package so it makes sense to offer all options. If this requires a patch first then if the files aren't available then maybe we shouldn't offer this option?  I'm wondering if something can be done in `TetraedgeMetaEngineDetection::toDetectedGame` to check for the availability before adding it to the list.  That may not be the best place to do it though, not sure how other engines handle this.

Either way it would be good to test that the game correctly falls back to English if the files are not available.  If it doesn't make sense to test for the data in the detector, we could do a toast on startup saying the chosen language is not available and it fell back to English, then translate that string.  There may even already be an equivalent string elsewhere in scummvm.

Reply:
Thank you, added a comment about the addtional patch... it does falls back to english if the files are not available
This is done by the check for the main language file: https://github.com/blooperz/scummvm/blob/9735eb9daa85f49398fbe3c6bb6fad0af28b3cd7/engines/tetraedge/game/application.cpp#L160-L172

(and currently also by the help file which is astonishing IMO)

Reply:
Looking at other engines, it seems they just print a warning (in console, not translated) that the language falling back to english automatically. (I think this behaviour makes sense)